osbs/http.py

```diff
@@ -34,6 +41,22 @@
 logger = logging.getLogger(__name__)
 
 
+class OSBSRetry(Retry):
+    def is_forced_retry(self, method, status_code, has_retry_after=False):
```

Comment:
Looks like the constructor for `Retry` takes `status_forcelist` which is used for setting the list of status codes which should trigger a retry: http://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util.retry